projects/canopy/karma.conf.js

```diff
@@ -27,6 +27,5 @@ module.exports = function (config) {
     autoWatch: true,
     browsers: ['Chrome'],
     singleRun: false,
-    restartOnFileChange: true,
```

Comment:
What's the reasoning here?

Reply:
Oh sorry, I thought I had written the comment for this - the jasmine tests kept running after doing ctrl + c which was incredibly annoying and removing this seemed to fix the issue

Reply:
Strange! OK cool :)